src/elements/types/ObjectProperty.js

```diff
@@ -53,6 +54,8 @@ function readKey(children) {
         children.isNode('Identifier')
     ) {
         return children.passNode();
+    } else if (children.isToken('Identifier')) {
+        return new Identifier([children.passToken()]);
```

Comment:
Yeah, that's because of the whole https://github.com/cst/cst/blob/ee17b3d601d931d045017c16bf0bbf7185da7b79/src/elementTree.js#L54 debacle :/

We can do two things here: either comment why this is how it is or we can come up with something